aztk/client/base/helpers/run.py

```diff
@@ -26,9 +26,10 @@ def cluster_run(base_operations, cluster_id, command, internal, container_name=N
                 command,
                 generated_username,
                 cluster_nodes,
-                ssh_key=ssh_key.exportKey().decode('utf-8'),
+                ssh_key=ssh_key.exportKey().decode("utf-8"),
                 container_name=container_name,
-                timeout=timeout))
+                timeout=timeout,
```

Comment:
Remove comma after timeout

Reply:
This is part of the linting definition so it is auto formatted.